rcl/test/rcl/test_arguments.cpp

```diff
@@ -1099,3 +1181,48 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_bad_allocs_copy
   }
   parsed_args.impl->allocator = saved_alloc;
 }
+
+TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_null_get_param_files) {
+  const std::string parameters_filepath1 = (test_path / "test_parameters.1.yaml").string();
+  const char * const argv[] = {
+    "process_name", "--ros-args", "--params-file", parameters_filepath1.c_str()
+  };
+  const int argc = sizeof(argv) / sizeof(const char *);
```

Comment:
@Blast545 FYI, no need to change anything: you can also do:
```suggestion
  const int argc = sizeof(argv) / sizeof(argv[0]);
```
for brevity.